packages/@o3r/application/schemas/cms.schema.json

```diff
@@ -31,9 +31,36 @@
       "type": "string",
       "description": "The relative path to the functional contents metadata folder to import"
     },
+    "agateEnabled": {
+      "type": "boolean",
+      "description": "Specify if Agate is enabled or not in the preview"
+    },
+    "baseNpmName": {
+      "type": "string",
+      "description": "if agateEnabled is set to true, this is used to set the proper Agate form type in the preview"
+    },
+    "features": {
+      "type": "object",
+      "description": "Contains features to be enabled or disabled",
+      "properties": {
+        "themingCssOverrideEnabled ": {
+          "type": "boolean",
+          "description": "Enable CSS override in theming"
+        },
+        "themingCssVariablesEnabled ": {
+          "type": "boolean",
+          "description": "Enable CSS variables in theming"
+        }
+      }
+    },
     "migrationScriptFolder": {
       "type": "string",
       "description": "The relative path to the migration scripts"
+    },
+    "migrationScriptVersionDependency": {
+      "type": "string",
+      "description": "Name of the dependency to get the current blueprint version. If the version is not found, version in the package.json will be used",
+      "default":"@refx/shared-common"
```

Comment:
I think there are 2 points here:
- Regarding the value of the default in this file: We actually need to avoid references to `refx`, `agate` or any specific internal project because of the engagement we had since we are open-source and to follow strategic decision.
- Regarding this default value on CMS side: as the migration script feature is not a feature dedicated to RefX it may lead to issue in other app